rider/src/main/resources/messages/UnityUIBundle.properties

```diff
@@ -8,6 +8,9 @@ dialog.unsaved.title.unity.scenes=Unsaved Unity Scenes
 dialog.unsaved.button.cancel=Cancel
 dialog.unsaved.button.commit.anyway=Commit Anyway
 
+attempt.to.commit.an.empty.folder.meta.files=Redundant meta files
```

Comment:
Should this start with a verb? Something like "Check redundant meta files"? It looks like the other options do.